src/components/App.js

```diff
@@ -1,19 +1,118 @@
 import React from 'react';
+import MenuComponent from "./MenuComponent";
+import Basket from './Basket';
 
 import '../styles/App.scss';
 
+
 class App extends React.Component {
-  constructor(){
+  constructor() {
     super();
+
+    this.state = {
+      menu : [],
+      currentOrder: [],
+      totalOrder: [],
+      totalPrice: 0,
+      showOrder: false
+    };
+    this.getFetch = this.getFetch.bind(this);
+    this.addOrder = this.addOrder.bind(this);
+    this.removeOrder = this.removeOrder.bind(this);
   }
 
-  render(){
-    return (
-      <div>
-        Delivereat app
-      </div>
-    )
+
+  componentDidMount() {
+    this.getFetch();
+  }
+
+
+  getFetch() {
+    const serverFetch = `http://localhost:8080/menu`
```

Comment:
it would better use relative url here `/menu` as that would allow the code to work when app is deployed to server with another url